src/anjaliSDec19/Assignment4/PatternPrinting/Patterna.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * 7.	Write programs to generate followingoutputs-
+a)	A
+AB
+ABC
+
+ */
+package anjaliSDec19.Assignment4.PatternPrinting;
+
+public class Patterna {
+
+	public static void main(String[] args) {
+		char[] ch = {'A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','R','S','T','U','V','W','X','Y','Z'};
```

Comment:
Instead of writing char array you can write
int ch = 65;
char ch = (char)ch;



int n = 5;
        for (int i = 65; i < 65 + n; i++) {
            for (int j = 65; j <= i; j++)
                System.out.print((char) i);
            System.out.println();
        }